lib/core/widgets/text_field_with_max.dart

```diff
@@ -0,0 +1,55 @@
+import 'package:deus/statics/my_colors.dart';
+import 'package:deus/statics/styles.dart';
+import 'package:flutter/material.dart';
+
+class TextFieldWithMax extends StatelessWidget {
+  final TextEditingController controller;
+  final double maxValue; // the maximum value that can be chosen
```

Comment:
- [ ] Dokumenationskommentare mit `///` darüber schreiben. 
Danach siehst du die Dokumentation auch, wenn du mit der Maus über maxValue hoverst (zumindest in VS Code, in Android Studio musst du evtl Strg+Q drücken).